jobbergate_cli/jobbergate_api_wrapper.py

```diff
@@ -131,11 +173,20 @@ def list_applications(self):
             verify=False).json()
         return application_list
 
-    def create_application(self, application_name):
-        f = open(self.application_config, "r")
-        data = json.loads(f.read())
+    def create_application(self, application_name, application_path, base_path):
+        data = self.application_config
         data['application_name'] = application_name
         data['application_owner'] = self.user_id
+
+        tar_name = f"{application_name}.tar.gz"
+
+        self.tardir(application_path, tar_name)
+
+        s3_key = base_path + str(self.user_id) + "/" + application_name + f"/{application_name}.tar.gz"
```

Comment:
this whole thing could be an f-string, also, you have `tar_name` above to be used in the fstring inplace of `f"/{application_name}.tar.gz"`

Reply:
committed this change to the PR